src/region.js

```diff
@@ -112,10 +112,12 @@ Marionette.Region = Marionette.Object.extend({
       // as it's a potentially-slow method
       var displayedViews = [];
 
-      var triggerBeforeAttach = showOptions.triggerBeforeAttach || this.triggerBeforeAttach;
-      var triggerAttach = showOptions.triggerAttach || this.triggerAttach;
+      var attachOptions = _.extend({
+        triggerBeforeAttach: this.triggerBeforeAttach,
+        triggerAttach: this.triggerAttach
+      }, showOptions);
```

Comment:
thanks!  first I tried checking explicitly for `showOptions.triggerAttach === false` but it became a pretty ugly logic statement.